addons/kruise-rollout/template.cue

```diff
@@ -0,0 +1,44 @@
+package main
+
+output: {
+	apiVersion: "core.oam.dev/v1beta1"
+	kind:       "Application"
+	metadata: {
+		name:      "kruise-rollout"
+		namespace: "vela-system"
+	}
+	spec: components: [{
+		name: "kruise-rollout"
+		type: "helm"
+		properties: {
+			repoType: "helm"
+			url:      "https://openkruise.github.io/charts/"
+			chart:    "kruise-rollout"
+			version:  "0.2.0"
+		}
+	}]
+}
+
+outputs: resourceTree: {
+	apiVersion: "v1"
+	kind:       "ConfigMap"
+	metadata: {
+		name:      "kruise-rollout-relation"
+		namespace: "vela-system"
+		labels: "rules.oam.dev/resources": "true"
+	}
+	data: rules: """
+		- parentResourceType:
+		    group: rollouts.kruise.io
+		    kind: Rollout
+		  childrenResourceType:
+		    - apiVersion: rollouts.kruise.io/v1alpha1
+		      kind: BatchRelease
+		- parentResourceType:
+		    group: rollouts.kruise.io
+		    kind: BatchRelease
+		  childrenResourceType:
+		    - apiVersion: apps/v1
+		      kind: Deployment
+		"""
```

Comment:
please use Marshal from CUE inner packages, refer to https://github.com/kubevela/catalog/blob/master/experimental/addons/clickhouse/template.cue#L22

please note you need to add Annotation for json format